numpy/random/mtrand/distributions.c

```diff
@@ -650,6 +650,9 @@ double rk_pareto(rk_state *state, double a)
 
 double rk_weibull(rk_state *state, double a)
 {
+    if (a == 0.0) {
```

Comment:
What happens if a<0? The [documentation](http://www.numpy.org/devdocs/reference/generated/numpy.random.weibull.html) says "Should be greater than 0". Is that accurate, or should it be changed to "Must be greater than 0" and we should error for negative values (raise an error in the pyx file)?

Reply:
Negative values are handled in `RandomState.weibull` in `mtrand.pyx`, so the C function is never called with negative values.  For example,

```
In [39]: np.random.weibull(a=-0.0)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-39-ee6de7ad2cfe> in <module>()
----> 1 np.random.weibull(a=-0.0)

mtrand.pyx in mtrand.RandomState.weibull()

ValueError: a < 0
```

Reply:
I guess "Should be greater than zero" should have been changed to "Must be nonnegative" back when the code was changed to allow `a=0`.  I can change that now.

Reply:
I just pushed an update to the docstring.